source/about/deployment_and_editions.rst

```diff
@@ -0,0 +1,164 @@
+
+Building on the features in Professional plan, it also offers:
+
+- `Shared channels with other Mattermost instances <https://docs.mattermost.com/onboard/shared-channels.html>`__
+- `Configurable playbook permissions, analytics dashboards and channel exports <>`_
+- `AD/LDAP Group synchronization <https://docs.mattermost.com/onboard/ad-ldap-groups-synchronization.html>`__
+- `Automated compliance exports and reporting <https://docs.mattermost.com/comply/compliance-export.html>`__
+- `Custom EMM providers via AppConfig <https://docs.mattermost.com/deploy/mobile-appconfig.html>`__
+- `Custom user terms of service <https://docs.mattermost.com/comply/custom-terms-of-service.html>`__
+- `ID-only push notifications <https://docs.mattermost.com/configure/configuration-settings.html#push-notification-contents>`__
+- `Global and custom retention policies for messages and files <https://docs.mattermost.com/comply/data-retention-policy.html>`__
+- `Custom admin roles <https://docs.mattermost.com/onboard/system-admin-roles.html>`__
+- `Granular audit logs with data export <https://docs.mattermost.com/comply/audit-log.html>`__
+- `High availability support with multi-node database deployment <https://docs.mattermost.com/scale/high-availability-cluster.html>`__
+- `Horizontal scaling through cluster-based deployment <https://docs.mattermost.com/scale/scaling-for-enterprise.html#cluster-based-deployment>`__
+- `Elasticsearch support for highly efficient database searches in a cluster environment <https://docs.mattermost.com/scale/elasticsearch.html>`__
+- `Advanced performance monitoring <https://docs.mattermost.com/scale/performance-monitoring.html>`__
+- `Eligibility for Premier Support add-on <https://mattermost.com/support/>`__
+
+See a complete list of features here <https://mattermost.com/pricing/>`__. 
+
+
+Mattermost Legacy Enterprise Editions
+--------------------------------------
+
+Mattermost introduced a new pricing and packaging structure on October 13, 2021. The packages listed below will be end-of-life October 31, 2023. 
+
+We are no longer selling these products to new customers. For existing customers, we highly recommend working with your Mattermost Account team to plan for a migration to our new packages, but we will honor existing pricing and features for renewals and expansions of E10/20 until September 30th, 2022. 
+
+Please contact our `sales team <https://mattermost.com/contact-us/>`__ with questions.
+
+Mattermost Enterprise Edition E10
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Mattermost E10 was offered as a commercial enterprise messaging solution for teams, groups, and departments working on multiple projects scaling from hundreds to thousands of users. E10 features are now available in Mattermost Professional <link to new package in doc>
```

Comment:
pending link needed

Reply:
I removed the link to be consistent with the next section on E20.